Description: Update a failing test case as per upstream
 Backport a small part of upstream commit, to fix 1 failing test case
Author: Ralf Stephan <ralf@ark.in-berlin.de>
Applied-Upstream: commit:3784da9e71986de287be1d6c3dca0f1d3473ead3
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage/functions/log.py
+++ b/sage/src/sage/functions/log.py
@@ -359,8 +359,10 @@
             1/6*pi^2
             sage: polylog(2,x^2+1)
             polylog(2, x^2 + 1)
-            sage: polylog(4,0.5)
-            polylog(4, 0.500000000000000)
+            sage: polylog(4, 1/2)
+            polylog(4, 1/2)
+            sage: polylog(4, 0.5)
+            0.517479061673899
 
             sage: f = polylog(4, 1); f
             1/90*pi^4
